app/js/tests/test_form_pages.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+var test = require( 'tape' ),
+	formPages = require( '../lib/form_pages' ),
+	Promise = require( 'promise' ),
+	pageSpy = {
+		display: false,
```

Comment:
I had the same though - I first saw it being used somewhere, and `display` seems like a method name. `isDisplayed` or `isVisible` are much nicer